infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/utils/LogReplicationConfigManager.java

```diff
@@ -184,9 +189,10 @@ private void openClientConfigTables() {
     /**
      * Generate LogReplicationConfig for all given sessions based on there replication model.
      *
-     * @param sessions set of sessions for which to generate config
+     * @param sessions set of sessions for which to generate config.
+     * @param registryTableOnly True if only update config from registry table.
      */
-    public void generateConfig(Set<LogReplicationSession> sessions) {
+    public void generateConfig(Set<LogReplicationSession> sessions, boolean registryTableOnly) {
         sessions.forEach(session -> {
                 switch (session.getSubscriber().getModel()) {
                     case FULL_TABLE:
```

Comment:
I found the config generation in listener is no longer needed, since it is now handled by SessionManager (when sessions are created) and updated by the sessions' reader/writer themselves. So I just removed the 2 places in listener.